core/src/com/unciv/logic/automation/UnitAutomation.kt

```diff
@@ -195,9 +195,13 @@ class UnitAutomation{
 
     class AttackableTile(val tileToAttackFrom:TileInfo, val tileToAttack:TileInfo)
 
-    fun getAttackableEnemies(unit: MapUnit, unitDistanceToTiles: PathsToTilesWithinTurn): ArrayList<AttackableTile> {
-        val tilesWithEnemies = unit.civInfo.viewableTiles
-                .filter { containsAttackableEnemy(it, MapUnitCombatant(unit)) }
+    fun getAttackableEnemies(
+            unit: MapUnit,
+            unitDistanceToTiles: PathsToTilesWithinTurn,
+            tiles: List<TileInfo>? = null
```

Comment:
Please change this to `tilesToCheckForEnemies` or something else that's more informative =)